tests/cpp/test_resize.cpp

```diff
@@ -4041,4 +4041,37 @@ TEST_F(ResizeTest, SliceSliceConcatConcat) {
   NVF_CHECK(ref.equal(cg_outputs[0]));
 }
 
+// manual scheduling that should have vectorized load on padded inputs.
+TEST_F(ResizeTest, VectorizePadLowering) {
```

Comment:
Should we have a test for vectorizing where without using pad?

Reply:
good call. almost forgot that we have `where` directly :face_with_head_bandage: